backup/queries_table_defs.go

```diff
@@ -194,10 +197,13 @@ func GetPartitionTableMap(connectionPool *dbconn.DBConn) map[uint32]PartitionLev
 			CASE WHEN p.partrelid IS NOT NULL AND c.relispartition = false THEN 0
 				ELSE rc.oid
 			END AS rootoid,
-			quote_ident(c.relname) AS name
+			quote_ident(c.relname) AS name,
+			objid is not NULL as isparentinextension
```

Comment:
I suggest to add table alias: `pd.objid`. It will be easier to understand which table this column is from.